tests/a11y/storybook.a11y.test.ts

```diff
@@ -1,8 +1,7 @@
 import expectNoAxeViolations from './expectNoAxeViolations';
 import { test } from '@playwright/test';
 import { stories } from '../../storybook-static/stories.json';
-
-const themes = ['core', 'healthcare' /* 'medicare' */];
+import * as Themes from '../../themes.json';
```

Comment:
Didn't we determine that `import *` inserts a `default` key? Why is it working?

Reply:
Updated to use the format used everywhere else. Something to do with the way I was using it in the other place. I can't remember exactly now.